modules/mono/editor/GodotTools/GodotTools/Build/BuildManager.cs

```diff
@@ -273,6 +285,56 @@ private static bool PublishProjectBlocking(BuildInfo buildInfo)
             return success;
         }
 
+        private static async Task<bool> BuildProjectAsync(BuildInfo buildInfo)
+        {
+            if (!File.Exists(buildInfo.Project))
+                return true; // No project to build.
+
+            if (_buildInProgress != null)
+            {
+                ShowBuildErrorDialog("Build already in progress!");
```

Comment:
If the build finishes while this dialog is open, you'll get this error:

```
ERROR: Attempting to parent and popup a dialog that already has a parent.
```

I don't think we need this dialog if we add some indication that there's a build ongoing instead.